modules/alloy/src/main/ts/ephox/alloy/positioning/layout/Bubble.ts

```diff
@@ -56,10 +59,15 @@ const allAlignments: Array<keyof BubbleAlignments> = [
   'top',
   'bottom',
   'left',
-  'right'
+  'right',
+
+  'inset'
 ];
 
-const nu = (width: number, yoffset: number, classes: BubbleAlignments): Bubble => {
+const nu = (width: number, yoffset: number, classes: BubbleAlignments, insetModifier: number = 1): Bubble => {
+  const insetWidth = width * insetModifier;
+  const insetYoffset = yoffset * insetModifier;
```

Comment:
I'd name the variable `yOffset` rather than `yoffset`. Just easier to read.

Reply:
Yeah, I don't like the names of either of these variables to be honest. I just kept it the same as it already was for consistency.

Reply:
Renamed them to `xOffset` and `yOffset` (as well as doing the same with the inset variables).